styles/fonts/details.json

```diff
@@ -0,0 +1,16 @@
+[
+    {
+        "name": "DejaVu Sans Mono",
+        "url": "https://dejavu-fonts.github.io/",
+        "license": "DejaVu Fonts license",
+        "license_url": "https://dejavu-fonts.github.io/License.html",
+        "license_file": "DejaVu Fonts License.txt"
+    },
+    {
+        "name": "DP Sans Mono",
+        "url": "https://www.pgdp.net/wiki/DP_Sans_Mono",
```

Comment:
Same as above -- if this page is not in the DP Official documentation, it should be protected.